moveit_ros/moveit_servo/include/moveit_servo/utilities.h

```diff
@@ -88,4 +89,23 @@ bool applyJointUpdate(rclcpp::Clock& clock, const double publish_period, const E
                       const sensor_msgs::msg::JointState& previous_joint_state,
                       sensor_msgs::msg::JointState& next_joint_state,
                       pluginlib::UniquePtr<online_signal_smoothing::SmoothingBaseClass>& smoother);
+
+/** \brief Converts a twist command from the command frame to the MoveGroup planning frame of the robot
+ * @param cmd The twist command
```

Comment:
Can you add more information for `cmd` and `servo_params` to the documentation here? Which twist command and why are the servo_params needed?